drivers/serial/Kconfig.renesas_ra2

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022-2024 MUNIC SA
```

Comment:
Consider integrating it into the existing uart_renesas_ra_sci.c.

Reply:
I'll try to do it. However it would probably be better to make this as a second iteration (as a separate PR).